### PR TITLE
B#pd 38 hand calibration gui plotjuggler to server

### PR DESCRIPTION
**Pull request by carebare47**
## Proposed changes

When using sr_gui_hand_calibration rqt gui on server&nuc/teleop, the button to plot a joint should ssh back from the nuc to the server and launch plotjuggler there.

Also sends constructed plotjuggler template from nuc to server

Breaks ability for this node to keep track of plotjuggler process and kill them when the node dies

Goes with this aurora PR: https://github.com/shadow-robot/aurora/pull/597

## Checklist

Before posting a PR ensure that from each of the below categories **AT LEAST ONE BOX HAS BEEN CHECKED**. If more than one category is applicable then more can be checked. Also ensure that the proposed changes have been filled out with relevant information for reviewers.

## Tests

- [ ] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure).
- [ ] Added/Modified automated and PhantomHand CI tests (if a new class is added (Python or C++), the interface of that class must be unit tested).
- [ ] Manually tested in simulation (if simulation specific or no hardware required to test the functionality). 
- [ ] Manually tested on hardware (if hardware specific or related).

## Documentation

- [ ] No documentation required to be added.
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc).
- [ ] Updated documentation (For changes to pre-existing features mentioned in the documentation).

### CODE REVIEW

File: advanced/sr_gui_hand_calibration/src/sr_gui_hand_calibration/hand_calibration.py [hunk 1]
```diff
@@ -32,6 +32,8 @@ class SrHandCalibration(Plugin):
     """
 
     def __init__(self, context):
+        rospy.logerr("###############################x1")
```
Comment (rnzenha-s) [inline advanced/sr_gui_hand_calibration/src/sr_gui_hand_calibration/hand_calibration.py]: remove?

**Review by dg-shadow — APPROVED** [review]

File: advanced/sr_gui_hand_calibration/src/sr_gui_hand_calibration/sr_hand_calibration_model.py [hunk 2]
```diff
@@ -217,6 +240,77 @@ def __init__(self, joint_name,
...
+            return f"docker exec -d {self._container_name} bash -c '{command}'"
+        return f"docker exec {self._container_name} bash -c '{command}'"
+
+    @staticmethod
+    def _prepend_source_ros(command):
+        return f"source /home/user/projects/shadow_robot/base/devel/setup.bash && {command}"
+
+    def _start_remote_plotjuggler(self, rosrun_command):
+        # Escape quotes
+        # rosrun_command = [x.replace("'", "\'") for x in rosrun_command]
```
Comment (rnzenha-s) [inline advanced/sr_gui_hand_calibration/src/sr_gui_hand_calibration/sr_hand_calibration_model.py]: remove comments

File: advanced/sr_gui_hand_calibration/src/sr_gui_hand_calibration/sr_hand_calibration_model.py [hunk 3]
```diff
@@ -217,6 +240,77 @@ def __init__(self, joint_name,
...
+        tmp_script_path = "/tmp/ssh_start_plotjuggler.sh"
```
Comment (rnzenha-s) [inline advanced/sr_gui_hand_calibration/src/sr_gui_hand_calibration/sr_hand_calibration_model.py:279]: Is this script allready being created/moved somewhere else?

**Review by rnzenha-s — APPROVED** [review]

File: advanced/sr_gui_hand_calibration/src/sr_gui_hand_calibration/sr_hand_calibration_model.py (hunk 3 above)
Comment (rnzenha-s) [reply advanced/sr_gui_hand_calibration/src/sr_gui_hand_calibration/sr_hand_calibration_model.py:279]: nevermind, I was blind

**Review by dg-shadow — APPROVED** [review]

**Pull request merged**